slowmovie.py

```diff
@@ -212,6 +211,7 @@ def find_subtitles(file):
 parser.add_argument("-s", "--start", type=int, help="start playing at a specific frame")
 parser.add_argument("-c", "--contrast", default=1.0, type=float, help="adjust image contrast (default: %(default)s)")
```

Comment:
Should we keep the contrast argument? Seems redundant now that Omni-EPD handles image adjustments.

Reply:
Would be one less arg to carry here. Might make sense to re-work the README a bit to explain values that can be adjusted or at least provide a link to the explanations. I'll work on something for that.